components/financial_assistance/app/views/financial_assistance/shared/_date_warning.html.erb

```diff
@@ -11,7 +11,8 @@
           <%= l10n('faa.start_date_warning', start_date_field_label: l10n('start_date'), current_year: current_year, next_year: next_year) %>
         </li>
         <li id=<%="end_date_warning_#{form_object_id}"%> class="hidden" data-cuke="end-date-warning">
-          <%= l10n('faa.end_date_warning', end_date_field_label: l10n('end_date')) %>
+          <b><%= l10n("faa.incomes.end_date_header") %></b>
+          <%= l10n("faa.incomes.end_date_reminder", end_date_field_label: l10n('end_date')) %>
```

Comment:
this key doesn't exist. I think we can keep the `faa.end_date_warning` that was removed on this line, and take out the "End Date" substring from that translation so it can be bolded in the markup as opposed to the translation value.